config/api.yml

```diff
@@ -770,6 +772,15 @@
       :post:
       - :name: assign
       - :name: unassign
+  :container_images:
+    :description: ContainerImages
+    :options:
+    - :subcollection
+    :verbs: *p
```

Comment:
it seems that not only would you want to scan them, but you would also want to be able to get them (ie `GET /api/providers/container_images`) after the scan.

Reply:
what should the `GET` return?

Reply:
I'm surprised that this works, as `p` isn't an alias for anything